Calculator/Calculator/Model/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by Erick on 2023/05/29.
+//
+
+struct CalculatorItemQueue {
+    private var queue = CalculatorItemLinkedList()
+    
+    var count: Int {
+        return queue.count
+    }
+    
+    var isEmpty: Bool {
+        return queue.head == nil
+    }
+    
+    mutating func enqueue(_ calculateItem: CalculateItem) {
+        queue.append(CalculatorItemNode(item: calculateItem))
+    }
```

Comment:
사바사 이긴 하지만, 저는 새로운 상수를 할당한 후에 그 상수를 대입하는 방식을 좀 더 선호합니다. 가독성이 더 좋거든요! 이러한 방법도 있으니 참고하시라는 의미로 보여드립니다 😉
```suggestion
    mutating func enqueue(_ calculateItem: CalculateItem) {
        let node = CalculatorItemNode(item: calculateItem)
        
        queue.append(node)
    }
```

Reply:
확실히 가독성 면에서 Wonbi가 보여주신 코드가 더 좋은 것 같습니다!!
상수에 할당한 후 대입하는 방식으로 수정하였습니다!